components/resources/commands.grdp

```diff
@@ -398,7 +398,7 @@
     Move group to new window
   </message>
   <message name="IDS_IDC_CLOSE_DUPLICATE_TABS">
```

Comment:
Hey, let's remove this so we don't have duplicate translations - we can point this command at the translation you added:

in `app/generate_command_metadata.py`, there's an `EXISTING_TRANSLATIONS` dictionary. If you add an entry for `'IDC_CLOSE_DUPLICATE_TABS': 'IDS_TAB_CXMENU_CLOSE_DUPLICATE_TABS'` you should be able to remove this.

Reply:
cool, fixed here: https://github.com/brave/brave-core/pull/22041/commits/3df4b35d1c105579c2bfb64a7aec291802b3721d